packages/core/upload/server/services/folder.js

```diff
@@ -23,6 +23,20 @@ const setPathAndUID = async folder => {
   });
 };
 
+const create = async (folderData, { user } = {}) => {
+  const { setPathAndUID } = getService('folder');
```

Comment:
You refactored the other ones into 
```suggestion
  const folderService = getService('folder');
  
  folderService.setPathAndUID()
```

Do you think you should follow this pattern here too?